neural_compressor/adaptor/mxnet_utils/util.py

```diff
@@ -24,7 +24,7 @@
 from enum import Enum
 from tempfile import TemporaryDirectory
 from neural_compressor.utils.utility import LazyImport
-from neural_compressor.model.model import MXNetModel as NCModel
+from neural_compressor.model.mxnet_model import MXNetModel as NCModel
```

Comment:
we need add MXNetModel and others to __init__.py for top namespace

Reply:
will improve it in bug fix period.